django_enum_choices/fields.py

```diff
@@ -47,13 +47,10 @@ def build_choices(self):
 
     def _calculate_max_length(self, **kwargs) -> int:
         max_length = kwargs.get('max_length')
+        max_choice_length = max([len(choice) for choice, _ in kwargs['choices']])
```

Comment:
You can omit the `[]` and just do `max(generator_expression)`

Reply:
Every time :(